src/framework/ui/internal/platform/windows/windowsplatformtheme.cpp

```diff
@@ -107,26 +80,84 @@ bool WindowsPlatformTheme::isFollowSystemThemeAvailable() const
     return m_buildNumber >= 17763; // Dark theme was introduced in Windows 1809
 }
 
-bool WindowsPlatformTheme::isDarkMode() const
+std::string WindowsPlatformTheme::themeCode() const
```

Comment:
I think would be cool, add a separate type for themeCode, like
```
using ThemeCode = std::string;
```

Reply:
I also thought that, let's do that!

Reply:
Done!